components/tutorial/tutorial_tip/tutorial_tip.tsx

```diff
@@ -14,10 +14,20 @@ import {t} from 'utils/i18n';
 
 import PulsatingDot from 'components/widgets/pulsating_dot';
 
+import * as Utils from 'utils/utils';
+
 import TutorialTipBackdrop, {Coords, TutorialTipPunchout} from './tutorial_tip_backdrop';
 
 const Preferences = Constants.Preferences;
-const TutorialSteps = Constants.TutorialSteps;
+const OnBoardingTutorialStep = Constants.TutorialSteps;
```

Comment:
Thanks for the suggestion, my editor starts showing wrong spelling errors as soon as I do ``onboarding`` instead of ``onBoarding`` which is annoying, and about starting with Capital letter I think we are Following that convention for ``Constant``